src/FileCache.php

```diff
@@ -168,7 +168,17 @@ public function delete(string $key): bool
             return true;
         }
 
-        return @unlink($file);
+        $result = @unlink($file);
+        // Check if error was because of file was already deleted by another process on high load
+        if ($result === false) {
+            $lastError = error_get_last();
+            if (str_ends_with($lastError['message'] ?? '', 'No such file or directory')) {
```

Comment:
```suggestion
            if (!is_file($file)) {
```

Maybe so more simplify?

Reply:
No, after unlink, this file can be already created by another process again